test/bats/test.bats

```diff
@@ -300,7 +301,46 @@ __namespace_exclusion_test() {
   run kubectl apply -f test/externaldata/dummy-provider/policy/examples/valid.yaml
   assert_success
 
+  # mutation test
+  run kubectl apply -f test/externaldata/dummy-provider/mutation/valid.yaml
+  assert_success
+
+  run kubectl run nginx --image=nginx --dry-run=server --output json
```

Comment:
We should wait until we know the mutator has been ingested before continuing, to avoid race conditions

https://github.com/open-policy-agent/gatekeeper/blob/97e9dd831ee5db1aaceb62982b6c3daa0ac53daa/test/bats/helpers.bash#L113-L140